frontend/src/metabase/palette/hooks/useRegisterShortcut.tsx

```diff
@@ -32,6 +35,14 @@ export const useRegisterShortcut = (
         return {
           ...shortcutDef,
           id,
+          perform: _.compose(
```

Comment:
Is `compose` needed? I might be confused but since we don't need to the output of the first function to be passed to the second function, could we make this dumber?

Reply:
🤷 not sure this is simpler

```ts
() => {
   trackSimpleEvent({
      event: "keyboard_shortcut_performed",
      event_detail: id,
    });
   perform();
}
```